src/mlpack/tests/random_forest_test.cpp

```diff
@@ -607,3 +607,117 @@ TEST_CASE("ExtraTreesAccuracyTest", "[RandomForestTest]")
 
   REQUIRE(accuracy >= 0.85);
 }
+
+/**
+ * Test ComputeAverageUniqueness.
```

Comment:
Can you make this and other comments a little more substantial and describe what is being tested, what is expected, and why?  I mean you don't have to go super overboard with it, but imagine that four years from now the test breaks because of some refactoring in `RandomForest`... so then the developer who made the change comes over to this test and needs to understand what is going on so they can debug whatever they broke. :)